kotlintest-core/src/main/kotlin/io/kotlintest/Project.kt

```diff
@@ -54,7 +54,7 @@ object Project {
   private val _projectlisteners = mutableListOf<ProjectListener>()
   private val _filters = mutableListOf<ProjectLevelFilter>()
   private var _specExecutionOrder: SpecExecutionOrder = LexicographicSpecExecutionOrder
-  private var writeSpecFailureFile: Boolean = true
+  private var writeSpecFailureFile: Boolean = false
```

Comment:
I'm not sure what this has to do with the feature

Reply:
That's on master too.